docs/monitoring.md

```diff
@@ -0,0 +1,79 @@
+# Monitoring
```

Comment:
Can we add it to the How To's section here? https://github.com/PlayFab/thundernetes/tree/main/docs/howtos